src/cl-giskard.lisp

```diff
@@ -50,58 +50,67 @@
 (defparameter *giskard-action-goal-type* "giskard_msgs/WholeBodyCommand")
 
 (defparameter *tf-listener* nil)
```

Comment:
Unlikely. Most of those parameters are initialized to strings that will never be modified by the code, and the reason I'm not using defconstant on them is because I get a weird error in the build otherwise.

As to the remaining parameters initialized with nil, these are ROS-related objects (tf listener, command topic publisher, and actionlib client) that are initialized exactly once, when first needed. The actionlib client should be persistent (so I don't think we should make one as a local variable in the send command action function), and in general these objects take some time to setup so it would be nice to avoid that time loss on most uses.